File: Java/src/jakopec/t2ucahurivanje/nikadaneraditiovako/Dokument.java
package jakopec.t2ucahurivanje.nikadaneraditiovako;

public class Dokument {

    int sifra;
    String brojDokumenta;
    double iznos;

    void obrada(){
        System.out.println("Obrađujem " + sifra + " - " + brojDokumenta + " - " + iznos);
    }

}

File: Kotlin/src/main/kotlin/t2ucahurivanje/Dokument.kt
package t2ucahurivanje

/**
 * @author Matija Skrletović
 **/

class Dokument(val sifra: Int, val brojDokumenta: String, val iznos: Double, izradio: Osoba?) {

    fun opisiMe(): String {
        return "$sifra - $brojDokumenta - $iznos"
    }

}